exporter/opensearchexporter/config.go

```diff
@@ -0,0 +1,7 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package opensearchexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter"
+
+type Config struct {
```

Comment:
This should have configuration structure at this point.  It may be wrong and need to change in the future, but we should have an idea at this point what the configuration should look like.  There should also be a README that outlines the component's purpose and usage, referring to the configuration as appropriate.

Reply:
Added content for configuration structure.